src/eureka/S3_data_reduction/s3_meta.py

```diff
@@ -156,10 +153,7 @@ def set_spectral_defaults(self):
         # Background parameters
         self.bg_hw = getattr(self, 'bg_hw')  # Require this parameter to be set
         self.bg_deg = getattr(self, 'bg_deg', 0)
-        # FINDME: I think bg_row_by_row, bg_dir, and isrotate all control
-        # the same thing...
         self.bg_row_by_row = getattr(self, 'bg_row_by_row', False)
-        self.bg_dir = getattr(self, 'bg_dir', 'CxC')
```

Comment:
I don't see it in Stage 1.

Reply:
I guess it doesn't need to be manually specified anywhere, but it is still used in S1_detector_processing/group_level.py, S3_data_reduction/background.py, S3_data_reduction/nircam.py, S3_data_reduction/plots_s3.py, and S3_data_reduction/s3_reduce.py.

Reply:
Oh, yeah... it's still necessary within the code.  It's just not a user-provided value.

Reply:
By "anywhere else" I meant anywhere else that it shouldn't be.

Reply:
Yes, but no. This solution avoids a bunch of if statements, which makes it cleaner.

Reply:
To provide a little more detail now that I'm at my laptop... `bg_row_by_row` is a boolean that tells the code whether or not to apply RxR BG subtraction before applying CxC BG subtraction, whereas `bg_dir` tells the BG subroutines whether we're currently performing RxR or CxC BG subtraction.  For NIRCam, initially `bg_dir = 'RxR'` then changes to `bg_dir = 'CxC'`.  I chose strings for `bg_dir` because it also allows for nice log statements, thus serving a duel purpose and removing `if` statements.